### ISSUE TITLE
Incorrect alignment of text with line and position of image inside pdf

### ISSUE THREAD

**Issue by arturasn** [yWorks/svg2pdf.js#159]
Some elements of the SVG are rendered incorrectly compared the html and pdf generated version of the svg2pdf.js.
1) Horizontal lines incorrectly aligned with APART1 text middle.
2) Html page start of image(y coordinate) seems to be much higher than in generated pdf even  y=0 is specified in the options.

For clearer understanding created small sample: [link](https://jsfiddle.net/36ep1o8r/)

**Comment by arturasn** [comment]
Examples where incorrect positioning is more likely seen: [link](https://jsfiddle.net/6vxm1sga/)

**Comment by yGuy** [comment]
Seems like `dominant-baseline="text-before-edge"` is the unsupported attribute. That's a known issue, I guess. So this seems like a duplicate of #80

**Comment by arturasn** [comment]
Seems to be `alignment-baseline="text-before-edge"` also unsupported, so it is not only an issue of `dominant-baseline`.
[link](https://jsfiddle.net/j9k48bLs/1/)

**Comment by yGuy** [comment]
Yes, see #109  and this comment: https://github.com/yWorks/svg2pdf.js/pull/114#pullrequestreview-307052649  - this is a missing feature in jspdf, already, as far as I can tell.

**Comment by arturasn** [comment]
As a work around currently I have changed svg directly:

1) `dominant-baseline="text-after-edge"` to `dominant-baseline="middle" dy="-0.3em"`
2) `dominant-baseline="text-before-edge"` to `dominant-baseline="middle" dy="0.3em"`
